src/views/Visualization.vue

```diff
@@ -1,46 +1,46 @@
 <template>
   <div id="visualization">
-    <h1>{{ title }}</h1>
+    <!-- <h1>{{ title }}</h1> -->
   </div>
 </template>
 
 <script>
 import * as d3Base from 'd3';
     export default {
         name: 'Visualization',
-        props: {
-            title: {
-                type: String,
-                default: process.env.VUE_APP_TITLE
-            }
-        },
-       data() {
-        return {
-          publicPath: process.env.BASE_URL, // allows app to find the files when on different deployment roots
-          d3: null,
-          viz_list: null,
-        }
-       },
-      mounted(){
-        this.d3 = Object.assign(d3Base);
-        this.loadData();  
-    },
-    methods:{
-      loadData() {
-        const self = this;
+      //   props: {
+      //       title: {
+      //           type: String,
+      //           default: process.env.VUE_APP_TITLE
+      //       }
+      //   },
+      //  data() {
+      //   return {
+      //     publicPath: process.env.BASE_URL, // allows app to find the files when on different deployment roots
+      //     d3: null,
+      //     viz_list: null,
+      //   }
+      //  },
+      //   mounted(){
+      //     this.d3 = Object.assign(d3Base);
+      //     this.loadData();  
+      // },
+      // methods:{
+      //   loadData() {
+      //     const self = this;
```

Comment:
Let's delete this if moved elsewhere. Per recent discussion, we may want to keep this here though to stage all the components and leave `App.vue` more blank